sdk/src/tx.rs

```diff
@@ -315,23 +327,18 @@ pub async fn broadcast_tx(
     )?;
 
     if response.code == 0.into() {
-        display_line!(
-            context.io(),
-            "Transaction added to mempool: {:?}",
-            response
-        );
+        display_line!(context.io(), "Transaction added to mempool.");
+        tracing::debug!("Transaction mempool response: {:?}", response);
```

Comment:
should we use the verbose debug print?
```suggestion
        tracing::debug!("Transaction mempool response: {response:#?}");
```